src/test/crimson/cbt/radosbench_4K_write.yaml

```diff
@@ -18,6 +18,7 @@ tasks:
           iops_avg: '(or (greater) (near 0.05))'
           iops_stddev: '(or (less) (near 0.05))'
           latency_avg: '(or (less) (near 0.05))'
+          cpu_cycles_per_op: '(or (less) (near 0.05))'
     monitoring_profiles:
       perf:
```

Comment:
I'm not sure whether the `perf` binary is already present at the host doing `make perf check`. Maybe we'll to install a packge. Still, that's fine.

Reply:
yeah, we can install `perf` in `ceph-perf-pull-requests` job.